### PR TITLE
Schema update for the project date field

### PR DESCRIPTION
**Pull request by mo-durvesh**
Validation on if a disabled radio field has been selected.

### CODE REVIEW

**Comment by mo-durvesh** [comment]
We may want to make this apart of the default schema for the radio field.

File: controllers/apply/under10k/fields/project-dates-next.js [hunk 1]
```diff
@@ -58,6 +59,9 @@ module.exports = {
                 cy: `Pryd hoffech chi gael yr arian os ydych chi'n cael eich dyfarnu?`,
             }),
             options: options(),
+            schema:  Joi.string().valid(
+                options().filter((option) => !has('disabled')(option.attributes)).map((option) => option.value)
```
Comment (mattandrews) [inline controllers/apply/under10k/fields/project-dates-next.js:63]: There's quite a lot going on in one line here, maybe we could make these named functions to make it clearer what's happening? Something like:

```
    get schema() {
        const enabledOptions = (option) => !has('disabled')(option.attributes);
        const optionValues = (option) => option.value;
        return options().filter(enabledOptions).map(optionValues);
    }
```

**Comment by mattandrews** [comment]
There's a [failing test](https://travis-ci.com/github/biglotteryfund/blf-alpha/builds/164235395#L477) here too!

**Comment by davidrapson** [comment]
Just pushed a fix for the tests, couple of our test cases were asserting the wrong thing.

**Pull request merged**